native/index.d.ts

```diff
@@ -0,0 +1,68 @@
+// <reference path="../react/react-native.d.ts" />
+
+declare module "styled-components/native" {
+    import * as ReactNative from "react-native";
+    import { StatelessComponent } from "react";
+
+    interface StyledFunction<T> {
+      (values: TemplateStringsArray): T;
+      (...values: any[]): T;
```

Comment:
uhm, I did this because I was getting an error when trying to use a template literal that had functions in it, like this:

![](http://s.patrick.wtf/MwlJNkYhTN/Screen-Shot-2016-11-26-16-31-38.png)

Not sure if there's a better way for it!